web/gui/dashboard_info.js

```diff
@@ -1590,6 +1590,164 @@ netdataDashboard.context = {
+        'other format errors, time-to-live exceeded, etc. '+
+        '<b>InAddrErrors</b> - invalid IP address or the destination IP address is not a local address and '+
+        'IP forwarding is not enabled. '+
+        '<b>InUnknownProtos</b> - unknown or unsupported protocol. '+
+        '<b>InTooBigErrors</b> - the size exceeded the link MTU. '+
+        '<b>InTruncatedPkts</b> - packet frame did not carry enough data. '+
+        '<b>InNoRoutes</b> - no route could be found while forwarding. '+
+        '<b>OutNoRoutes</b> - no route could be found for packets generated by this host.</p>'
+    },
+
+    'ipv6.udppackets': {
+        info: 'The number of transferred UDP packets.'
+    },
+
+    'ipv6.udperrors': {
+        info: '<p>The number of errors encountered during transferring UDP packets.</p>'+
+        '<b>RcvbufErrors</b> - receive buffer is full. '+
+        '<b>SndbufErrors</b> - send buffer is full, no kernel memory available, or '+
+        'the IP layer reported an error when trying to send the packet and no error queue has been setup. '+
+        '<b>InErrors</b> - that is an aggregated counter for all errors, excluding <b>NoPorts</b>. '+
+        '<b>NoPorts</b> - no application is listening at the destination port. '+
+        '<b>InCsumErrors</b> - a UDP checksum failure is detected. '+
+        '<b>IgnoredMulti</b> - ignored multicast packets.'
+    },
+
+    'ipv6.udplite': {
+        info: 'The number of transferred UDP-Lite packets.'
+    },
+
+    'ipv6.udplite_errors': {
+        info: '<p>The number of errors encountered during transferring UDP-Lite packets.</p>'+
+        '<p><b>RcvbufErrors</b> - receive buffer is full. '+
+        '<b>SndbufErrors</b> - send buffer is full, no kernel memory available, or '+
+        'the IP layer reported an error when trying to send the packet and no error queue has been setup. '+
+        '<b>InErrors</b> - that is an aggregated counter for all errors, excluding <b>NoPorts</b>. '+
+        '<b>NoPorts</b> - no application is listening at the destination port. '+
+        '<b>InCsumErrors</b> - a UDP checksum failure is detected.</p>'
+    },
+
+    'ipv6.mcast': {
+        info: 'Total IPv6 multicast traffic.'
+    },
+
+    'ipv6.bcast': {
+        info: 'Total IPv6 broadcast traffic.'
+    },
+
+    'ipv6.mcastpkts': {
+        info: 'Total transferred IPv6 multicast packets.'
+    },
+
+    'ipv6.icmp': {
+        info: '<p>The number of transferred ICMPv6 messages.</p>'+
+        '<p><b>Received</b>, <b>Sent</b> - ICMP messages which the host received and attempted to send. '+
+        'Both these counters include errors.</p>'
+    },
+
+    'ipv6.icmpredir': {
+        info: 'The number of transferred ICMPv6 Redirect messages. '+
+        'These messages inform a host to update its routing information (to send packets on an alternative route).'
+    },
+
+    'ipv6.icmpechos': {
+        info: 'The number of ICMPv6 Echo messages.'
```

Comment:
Is `Echo` in uppercase expected? Or did we have typo error?

Reply:
Yes, all control name messages names start with an uppercase letter. See https://www.iana.org/assignments/icmpv6-parameters/icmpv6-parameters.xhtml

Reply:
Thank you!